### DISCUSSION TITLE
Mac OS, can you remove the menu bar?

### DISCUSSION THREAD

**Discussion by benfrain** [wezterm/wezterm#3035] [Q&A] [answered]
Is there a way to disable the menu bar for WezTerm in MacOS?

Wez looks like this:

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/1318466/215606381-03cc1aa9-0516-41a0-a55c-1f10eabfbf92.png">

With Kitty you can configure it like this (note the missing grey menu bar):

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/1318466/215606581-f4daa3a9-5f8b-434c-87d3-c615fcbdaa60.png">

Can't find it in options — is it possible?

**Comment by wez** [comment, marked as answer]
That's the title bar. Take a look at https://wezfurlong.org/wezterm/config/lua/config/window_decorations.html

**Reply by benfrain** [reply]
Fantastic! Thanks 🙏

**Comment by benfrain** [comment]
I think there might be a bug. With the `window_decoration = "NONE"` setting, the window does not seem to know its own size. For example I use Rectangle Pro as my Window manager, and when I try and move WezTerm to one side of other of my screen, it is not sized correctly. Without that option it works as expected.

Only just using Wezterm today so not sure if I have something wrong. If this sounds like a bug I will open an issue.

**Reply by wez** [reply]
You probably want `window_decorations = "RESIZE"`

**Reply by benfrain** [reply]
Perfect, thanks
